plugins/src/main/java/org/smartregister/fhir/gateway/plugins/BaseEndpoint.java

```diff
@@ -11,7 +11,7 @@
 import org.hl7.fhir.r4.model.Resource;
 
 public abstract class BaseEndpoint extends HttpServlet {
-    public Bundle createBundle(List<Resource> resourceList) {
+    public static Bundle createBundle(List<Resource> resourceList) {
```

Comment:
Since these are now being used by multiple classes lets move them to the Utils class aforementioned